header_layout.ml

```diff
@@ -22,4 +22,12 @@ let header ?(page_title = "Mollymawk") ~icon () =
           ~href:"https://unpkg.com/aos@2.3.1/dist/aos.css" ();
         script ~a:[ a_src "https://unpkg.com/aos@2.3.1/dist/aos.js" ] (txt "");
         link ~rel:[ `Icon ] ~href:icon ();
+        script
+          ~a:
+            [
+              a_defer ();
+              a_src
+                "https://cdn.jsdelivr.net/npm/alpinejs@3.x.x/dist/cdn.min.js";
```

Comment:
(I'm as well unsure why aos is needed, but I'm fine with your choices)

Reply:
thanks for the explanation, no need to remove aos. it would help me to have a comment in the code a la "aos is animate-on-scroll, used for the home page and make cards bouncy" and "alpine makes multiselect nice"